ports/nrf/common-hal/storage/__init__.c

```diff
@@ -35,8 +35,9 @@
 extern volatile bool mp_msc_enabled;
 
 void common_hal_storage_remount(const char *mount_path, bool readonly) {
-    if (strcmp(mount_path, "/") != 0)
+    if (strcmp(mount_path, "/") != 0) {
         mp_raise_OSError(MP_EINVAL);
+    }
```

Comment:
I've been kinda not using brackets for single line ifs, not sure if we need to be adding them back

Reply:
I like them but would rather spend the brainpower getting `clang-format` going.